package.json

```diff
@@ -1,54 +1,61 @@
 {
   "name": "mc-components",
   "version": "0.2.9",
-  "main": "./dist/index.js",
   "author": "Masterclass <engineering@masterclass.com>",
   "homepage": "https://github.com/yankaindustries/mc-components",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/yankaindustries/mc-components/issues"
   },
-  "dependencies": {
-    "bootstrap": "^4.1.1",
-    "classnames": "^2.2.5",
-    "react-slick": "^0.23.1",
-    "slick-carousel": "^1.8.1"
-  },
+  "main": "./dist/main.js",
+  "module": "./dist/module.js",
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "yarn build:js && yarn build:assets && yarn build:styles && yarn build:css-vendor",
+    "build": "yarn build:js && yarn build:assets && yarn build:styles",
     "build:assets": "cpx -p './src/assets/**/*' ./dist/assets",
-    "build:js": "NODE_ENV=production babel ./src --out-dir ./dist --ignore spec.js,test.js,stories.js",
-    "prebuild:css": "node-sass src/styles/index.scss dist/styles/css/mc-components.css",
-    "build:css": "postcss -r dist/styles/css/mc-components.css",
-    "build:css-vendor": "node-sass src/styles/libraries/_libraries.scss dist/styles/css/vendor.css && postcss -r dist/styles/css/vendor.css",
-    "build:scss": "cpx -p './src/styles/**/*.scss' ./dist/styles/scss",
-    "build:styles": "yarn build:scss && yarn build:css",
+    "build:js": "rollup -c",
+    "build:css": "node-sass src/styles/index.scss | postcss > dist/styles.css",
+    "build:css-vendor": "node-sass src/styles/libraries/_libraries.scss | postcss > dist/vendor.css",
+    "build:styles": "yarn build:css && yarn build:css-vendor",
     "start": "start-storybook -p 9001 -c .storybook",
     "deploy-storybook": "storybook-to-ghpages",
     "prepublishOnly": "yarn build",
     "lint:js": "eslint src --color",
     "lint:style": "stylelint \"src/styles/**/*.scss\" || echo \"Style lint failed.\"",
     "lint": "yarn lint:js && yarn lint:style",
-    "precommit": "lint-staged",
```

Comment:
I'm not sure I agree with this one - I like having linting done on commit because otherwise you need to remember to run it manually before pushing a PR.

Reply:
CircleCI will still lint for you, but if you quickly want to save your work the linter stops you from being able to commit and switch to something else.  This, IMO, is a blocker.

https://ourmasterclass.slack.com/archives/GBC5JV6HF/p1532498175000023

Reply:
If we can figure out how to have the linter show you the errors, but not block you from committing, I'm fine with that.  But blocking is 👎

Reply:
you can use `--no-verify` to commit without linting

Reply:
Good to know.